CMakeLists.txt

```diff
@@ -20,22 +20,60 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # Get verbose output from cmake generation and build steps
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
+# Set target OS platform
+if (CMAKE_SYSTEM_NAME MATCHES Linux)
+    set(DO_PLATFORM_LINUX 1)
```

Comment:
>DO_PLATFORM_LINUX [](http://example.com/codeflow?start=8&length=17)

Thought I'd simplify this while fixing the build issue. Easier/more correct to check for a definition of var/macro instead of its value.